### ISSUE TITLE
Implement pinball loss for quantile regression

### ISSUE THREAD

**Issue by tiepvupsu** [tiepvupsu/tabml#260]

**Comment by tiepvupsu** [comment]
Ref: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_pinball_loss.html#sklearn.metrics.mean_pinball_loss

**Comment by tiepvupsu** [comment]
Added in https://github.com/tiepvupsu/tabml/pull/261.
